spec/models/manageiq/providers/embedded_ansible/automation_manager/job_spec.rb

```diff
@@ -65,6 +73,7 @@
         expect(subject.ems_ref).to eq(the_raw_job.id)
         expect(subject.status).to  eq(the_raw_job.status)
         expect(subject.parameters.first).to have_attributes(:name => 'param1', :value => 'val1')
+        expect(subject.authentications).to eq([machine_credential, cloud_credential, network_credential])
```

Comment:
This should be `match_array` instead of `eq`.  This way a change in order should not affect the test.